pkg/instaclustr/client.go

```diff
@@ -145,6 +146,10 @@ func (c *Client) UpdateOpenSearch(id string, o models.OpenSearchInstAPIUpdateReq
 		return err
 	}
 
+	if strings.Contains(string(body), ClusterIsNotInStateToResize) {
+		return ClusterIsNotReadyToResize
+	}
```

Comment:
I think it's better to use status Conflict

Reply:
Status Conflict can be error other from "not ready to resize"